src/main/resources/static/api.yaml

```diff
@@ -2172,3 +2242,33 @@ components:
       properties:
         refreshToken:
           type: string
+    OtpVerificationRequest:
+      type: object
+      properties:
+        username:
+          type: string
+    OtpVerificationResponse:
+      type: object
+      properties:
+        sessionId:
+          type: string
+    OtpPermitRequest:
+      type: object
```

Comment:
use required attributes mark what are essential.

Reply:
Done.

Reply:
Do we need username as part of the request?